editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/ThemeAndLogoForm.tsx

```diff
@@ -0,0 +1,113 @@
+import Link from "@mui/material/Link";
+import { getContrastRatio, useTheme } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import { Team, TeamTheme } from "@opensystemslab/planx-core/types";
+import { useFormik } from "formik";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React, { useEffect, useState } from "react";
+import ColorPicker from "ui/editor/ColorPicker";
+import InputDescription from "ui/editor/InputDescription";
+import InputRow from "ui/shared/InputRow";
+import InputRowItem from "ui/shared/InputRowItem";
+import InputRowLabel from "ui/shared/InputRowLabel";
+import PublicFileUploadButton from "ui/shared/PublicFileUploadButton";
+
+import { DesignPreview, SettingsForm } from ".";
+
+type FormValues = Pick<TeamTheme, "primaryColour" | "logo">;
+
+export const ThemeAndLogoForm: React.FC<{ team: Team }> = ({ team }) => {
+  const theme = useTheme();
+
+  useEffect(() => {
+    setInitialValues({
+      primaryColour: team.theme?.primaryColour,
+      logo: team.theme?.logo,
+    })
+  }, [team])
+  
+  const [initialValues, setInitialValues] = useState<FormValues>({
+    primaryColour: "",
+    logo: "",
+  })
```

Comment:
This feels a bit awkward but was the best way I could get the form to handle async initial values from outside the component. Without this the form reverted to its initial state on reset (even after a successful "Save").